src/utils/createXArray.ts

```diff
@@ -16,6 +14,11 @@ export function createXArray(
      * end value of range
      * @default 1 */
     to?: number;
+    /**
+     * step if "to" not defined
+     * @default null
```

Comment:
Please add many testcases using the 'step' to see if it works correctly and is intuitive.